src/assets/i18n/en.json

```diff
@@ -119,7 +121,14 @@
     "systemSettings": "system settings",
     "exploreTrajectoryTitle": "Explore your trajectory",
     "showNotifications": "Show Notifications",
-    "loading": "Waiting…"
+    "loading": "Waiting…",
+    "agreement_start": "I agree with the ",
+    "agreement_end": "terms and privacy policy.",
```

Comment:
As above: You're mixing camelCase and snake_case here. I think we're mostly using camelCase in these files.